agreement/state_machine_test.go

```diff
@@ -150,6 +150,15 @@ func (t ioTrace) ContainsFn(compareFn func(b event) bool) bool {
 	return false
 }
 
+func (t ioTrace) countAction() (count int) {
```

Comment:
I don't see where you use this method. did I miss anything ?

Reply:
I use this to verify that the number of actions emitted by a player receiving a messageEvent lines up with what if should be, ie
```
			require.Equalf(t, trace.countAction(), 1, "Plyaer should not emit extra actions, player: %v, event: %v", playerN, eventN)
```

Reply:
sounds good, thanks.